lib/galaxy/model/__init__.py

```diff
@@ -3915,16 +3917,26 @@ def extra_files_path_name(self):
     def _extra_files_rel_path(self):
         return self._extra_files_path or self.extra_files_path_name
 
-    def _calculate_size(self):
+    def _calculate_size(self) -> int:
         if self.external_filename:
             try:
                 return os.path.getsize(self.external_filename)
             except OSError:
                 return 0
-        else:
+        elif self.object_store:
             return self.object_store.size(self)
+        else:
+            return 0
```

Comment:
Shouldn't that be an error condition ?

Reply:
Addressed in a rebase by using `assert self.object_store` instead.